features/step_definitions/digital_summary_steps.rb

```diff
@@ -0,0 +1,18 @@
+Given(/^the customer (.*) a digital appointment summary$/) do |requested|
```

Comment:
We spoke about this in person but to summarise: this feels like a unit test, rather than something the end user could observe.